xds/src/main/java/io/grpc/xds/InternalXdsAttributes.java

```diff
@@ -81,9 +79,14 @@
    * Filter chain match for network filters.
    */
   @Grpc.TransportAttr
-  static final Attributes.Key<AtomicReference<FilterChainSelector>>
-          ATTR_FILTER_CHAIN_SELECTOR_REF = Attributes.Key.create(
-          "io.grpc.xds.InternalXdsAttributes.filterChainSelectorRef");
+  static final Attributes.Key<FilterChainSelectorManager>
+          ATTR_FILTER_CHAIN_SELECTOR_MANAGER = Attributes.Key.create(
+          "io.grpc.xds.InternalXdsAttributes.filterChainSelectorManager");
+
+  /** Grace time to use when draining. Null for an infinite grace time. */
+  @Grpc.TransportAttr
+  static final Attributes.Key<Long> ATTR_DRAIN_GRACE_NANOS =
+      Attributes.Key.create("io.grpc.xds.InternalXdsAttributes.filterChainSelectorManager");
```

Comment:
Is it a typo? MANAGER and GRACE_NANOS key names are the same

Reply:
Yeah, I copied the other line to avoid retyping the "io.grpc.xds.InternalXdsAttributes", and then got sidetracked on reflowing the line and such.

Note that this would not cause any behavior issues. These strings are debug strings, just used in toString(). They aren't actually used to identify the keys.